src/test/java/seedu/address/logic/commands/CloseCommandTest.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+
+public class CloseCommandTest {
+
+    private Model model = new ModelManager();
+    @Test
+    public void execute_close_success() {
+        CloseCommand closeCommand = new CloseCommand();
+        CommandResult commandResult = closeCommand.execute(model);
+
+        assertEquals(CloseCommand.MESSAGE_SUCCESS, commandResult.getFeedbackToUser());
+        assertFalse(commandResult.isShowHelp());
+        assertFalse(commandResult.isExit());
+    }
```

Comment:
Perhaps you can add one more test to check if when close is used when the detailed panel is not shown, that nothing happens?

Reply:
Testing on GUI quite a big change, have opened a new issue to further add that #120